Nodejs/Product/Nodejs/Options/NodejsGeneralOptionsControl.cs

```diff
@@ -66,6 +66,11 @@ public partial class NodejsGeneralOptionsControl : UserControl {
             _waitOnNormalExit.Checked = page.WaitOnNormalExit;
             _editAndContinue.Checked = page.EditAndContinue;
             _checkForLongPaths.Checked = page.CheckForLongPaths;
+            _webkitDebugger.Checked = page.UseWebKitDebugger;
+
+#if !DEV15
+            _webkitDebugger.Enabled = false;
```

Comment:
So does this option appear at all on < Dev15? Or does it appear and is disabled?